CONTRIBUTING.md

```diff
@@ -40,3 +40,13 @@ refer to [this section][contributing-bug-reports] and [open an issue][issue temp
 [internals]: https://internals.rust-lang.org
 [rust-discord]: http://discord.gg/rust-lang
 [rust-zulip]: https://rust-lang.zulipchat.com
+
+## External LLVM Version Compatibility
+
+To build `rustc`, it is important to know which versions of LLVM are supported. As of Rust 1.82, the following external LLVM versions are supported:
+
+- **Minimum Supported Version**: LLVM 17
```

Comment:
This is still not the right version, we don't support LLVM 17 anymore. Where are you getting this information?

Reply:
![image](https://github.com/user-attachments/assets/8841a96b-f138-4a90-bdf7-858a5e7bcfbb)

Reply:
Link?

Reply:
Where is this?

Reply:
[v](https://www.tublian.com/project/724712/131588)

Reply:
https://www.tublian.com/project/724712/131588

Reply:
https://github.com/rust-lang/rust/issues/131588 ?

Reply:
mybe